Monika After Story/game/script-topics.rpy

```diff
@@ -6365,3 +6365,28 @@ label monika_hair_down:
     $ unlockEventLabel("monika_hair_ponytail")
 
 ##### End monika hair topics
+
+init 5 python:
+    addEvent(Event(persistent.event_database,eventlabel="backpacking",category=['misc'],prompt="Backpacking",random=True))
+
+label backpacking:
```

Comment:
lol yes, yes you do, I'm a dope. I just copied another topic for the formatting and accidentally wrote over the entire label instead of leaving the monika_ part in. Oops.